syntax = "proto3";

package user;

service User {
  rpc GetInfo (UserInfoRequest) returns (UserInfoReply) {}
}

message UserInfoRequest {
  string access_token = 1;
}

message UserInfoReply {
  string id = 1;
  string login = 2;
  string email = 3;
  string first_name = 4;
  string last_name = 5;
  repeated string rights = 6;
}